portfolio/src/main/webapp/index.html

```diff
@@ -7,11 +7,12 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
     <title>MM Portfolio</title>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBjTF_cxIVfkP_Z9NeXTcUJFNHiLjeYDhk"></script>
+    <script async defer
```

Comment:
OOC - why is `async defer` required here?

Reply:
async defer ensures that the callback is only invoked once the google api is loaded...is that in line with the style guidelines or should I continue to use onload? I thought this way looked cleaner, but it could definitely make the header information more dense